assets/component-facets.css

```diff
@@ -566,7 +566,6 @@ a.active-facets__button.focused .active-facets__button-inner,
 }
 
```

Comment:
This is still happening in the quantity input: https://screenshot.click/01-05-auu32-gr5ce.png (the element is not in focus), but the colors are not staying in the container

Reply:
This probably should have been fixed separately, but the fix was easy enough. Should be resolved.